sync/src/main/kotlin/gropius/sync/AbstractSync.kt

```diff
@@ -323,6 +330,79 @@ abstract class AbstractSync(
         updater.save(collectedSyncInfo.nodeRepository)
     }
 
+    /**
+     * Lookup a state by its ID
+     * @param id ID of the state
+     * @return State with the given ID or null if given null
+     */
+    private suspend fun lookupState(id: String?): IssueState? {
+        if (id != null) {
+            return collectedSyncInfo.neoOperations.findById<IssueState>(id)!!
+        }
+        return null
+    }
+
+    private suspend fun integrateLabelToState(
```

Comment:
docs

Reply:
done in earlier commit (?) maybe a0cfce13d00e30d0323c4845dab70e6c8d39ea0c